examples/cdi-produces-disposes/README_pt.adoc

```diff
@@ -0,0 +1,291 @@
+= CDI Produces Disposes
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+Esté exemplo demostra como as anotaçõẽs @Produces e @Disposes trabalham. Um LogFactory cria uma instância do LogHandler
+dependendo de um atributo "type". Para o propósito deste exemplo, o type é hard-coded para especificar um valor.
+Uma implementação Logger deve contér uma lista de LogHandlers. Teremos três implementações da interface LogHandler.
+
+* Uma DatabaseHandler
+* Uma FileHandler
+* Uma ConsoleHandler
+
+O DatabaseHandler aparentemente gravar os logs em um banco de dados. O FileHandler aparentemente gravar os logs em um arquivo.
```

Comment:
"aparentemente gravar"

Reply:
What is wrong here?

Reply:
I will answer in Portuguese so that we do not have doubts of interpretation. Em português seria a conjugação do verbo, que no caso seria "aparentemente grava". Mas em ingles o autor usa o verbo auxiliar "would", que nesse caso é usado para indicar possibilidade, possibilidade de gravar os logs em um banco de dados, então seria "aparentemente gravaria os logs em um banco de dados".

Reply:
cool, thanks .

Reply:
done.